deploy/README-multicloud.md

```diff
@@ -0,0 +1,100 @@
+# Configuring Service Assurance Framework for Multiple Clouds
+
+Multiple Openstack clouds can be configured to target a single instance of SAF.
```

Comment:
s/SAF/Service Assurance Framework (SAF)/

Usually when using an acronym it's called out in full the first time, then defined and used for the rest of the document.